exporter/datadogexporter/go.mod

```diff
@@ -3,9 +3,18 @@ module github.com/DataDog/opentelemetry-collector-contrib/exporter/datadogexport
 go 1.15
 
 require (
+	github.com/DataDog/datadog-agent v0.0.0-20200417180928-f454c60bc16f
+	github.com/DataDog/viper v1.8.0 // indirect
+	github.com/census-instrumentation/opencensus-proto v0.3.0
+	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575 // indirect
+	github.com/gogo/protobuf v1.3.1
+	github.com/klauspost/compress v1.10.10
 	github.com/stretchr/testify v1.6.1
 	github.com/zorkian/go-datadog-api v2.29.0+incompatible // indirect
 	go.opentelemetry.io/collector v0.11.1-0.20201006165100-07236c11fb27
```

Comment:
Believe this should match the current v0.12.0

Reply:
👍 updated from master so should be good to go here, pr was a bit stale